accurics-aws-tf-sample.tf

```diff
@@ -126,9 +126,21 @@ resource "aws_s3_bucket" "accurics-test-s3bucket1" {
   }
 
   tags = {
-    Name = format("%s-s3bucket1", var.acqaPrefix)
+    Name         = format("%s-s3bucket1", var.acqaPrefix)
     ACQAResource = "true"
-    Owner = "Accurics"
+    Owner        = "Accurics"
+  }
```

Comment:
**Accurics remediation**
 
**[HIGH]: S3 bucket accessible to all authenticated users.**
 
**Vulnerability**
S3 bucket Access is allowed to all AWS Account Users.
 
**Remediation**
S3 bucket Access should be limited to specific users and roles.

source: accurics-app